### PR TITLE
RPPL-2419: deleted the firebolt-open-rpc.json from Ripple, necessary erro…

### PR DESCRIPTION
**Pull request by nnaveen979**
…r handling updated.

## What

Remove: firebolt-open-rpc.json file from Ripple.

## Why

This will reduce the ripple filesystem footprint by approx 257kb-501kb

## How

deleted the firebolt-open-rpc.json from Ripple, by environment variables , build system still access this file. full details updated in Ripple README file.

## Test

firebolt-open-rpc.json file deleted and run Ripple, it's referring the expected path.

## Checklist

- [+] I have self-reviewed this PR
- [ ] I have added tests that prove the feature works or the fix is effective

### CODE REVIEW

**Review by pahearn73 — COMMENTED** [review]
We should be attempting to load firebolt-open-rpc.json exactly the same way we do for manifests, e.g. for device manifest we have try_manifest_files() which tries to load from various sources e.g. load_from_env(), load_from_home(), load_from_etc(). Create these same functions, do it the same way. See [here](https://github.com/rdkcentral/Ripple/blob/main/core/main/src/bootstrap/manifest/device.rs#L25).

Your unit tests are failing because the path to firebolt-open-rpc.json is empty. That's probably because you haven't added:

export FIREBOLT_OPEN_RPC=${workspace_dir}/target/openrpc/firebolt-open-rpc.json

...to run.sh and runCI.sh.

File: core/main/src/state/openrpc_state.rs [hunk 1]
```diff
@@ -125,55 +125,20 @@ impl OpenRpcState {
         }
     }
 
-    fn load_firebolt_open_rpc() -> FireboltVersionManifest {
```
[the remark this answers is not in the document]
Comment (brendanobra) [reply core/main/src/state/openrpc_state.rs:128]: run.ch and runCI.sh are not used in the unit test step in the github action that runs the ci for PRs (https://github.com/rdkcentral/Ripple/blob/main/.github/workflows/Build.yml) so this ^^^ will not work. The issue appears to be that  that there is a coupling between the unit tests and `firebolt-open-rpc.json`, which does not exist in Ripple (and only seems to exist in a closed source repo). I think the best way to proceed is to either 1) add a `firebolt-open-rpc.json` into Ripple (which may have to be maintained, not sure of the details) or 2) add mock/test gated compilation for different behavior during test in openrpc_state.rs (this is probably better, but I have not looked too deeply into it yet)

File: .github/workflows/Build.yml [hunk 2]
```diff
@@ -41,7 +41,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions-rust-lang/setup-rust-toolchain@v1
-      - run: cargo test
+      - run: ./ripple test
```
Comment (brendanobra) [inline .github/workflows/Build.yml:44]: As we discussed, I don't think this is best practice  - the unit tests should run with only `cargo test`, and not require a wrapper to setup the environment

Comment (nnaveen979) [reply .github/workflows/Build.yml:44]: I agree, will remove these changes.

**Review by pahearn73 — APPROVED** [review]

**Review by brendanobra — APPROVED** [review]

**Review by tfarz123 — APPROVED** [review]

**Pull request merged**